mcdreforged/cli/cli_entry.py

```diff
@@ -45,6 +47,17 @@ def entry_point():
 	parser_pack.add_argument('-o', '--output', help='The output directory to store the zipped plugin, default: current directory', default='.')
 	parser_pack.add_argument('-n', '--name', help='A specific name to the output zipped plugin file. If not given the metadata specific name or a default one will be used', default=None)
 
+	parser_workspace = subparsers.add_parser('init_plugin', help='Initialize a plugin workspace')
+	parser_workspace.add_argument('-p', '--path',         help='The plugin path, default: current directory',        default=None)
+	parser_workspace.add_argument('-i', '--id',           help="The plugin id",                                      default=None)
+	parser_workspace.add_argument('-n', '--name',         help="The plugin name, default: as same as the plugin id", default=None)
+	parser_workspace.add_argument('-d', '--description',  help="The plugin description",                             default=None)
+	parser_workspace.add_argument('-a', '--author',       help="The plugin author(s), split with ':'",               default=None)
+	parser_workspace.add_argument('-l', '--link',         help="The plugin link",                                    default=None)
+	parser_workspace.add_argument('-r', '--resources',    help="The plugin resources files, split with ':'",         default=None)
+	parser_workspace.add_argument('-e', '--entrypoint',   help="The plugin's entry point",                           default=None)
+	parser_workspace.add_argument('-A', '--archive-name', help="The plugin's archive name",                        default=None)
```

Comment:
You are trying to align the parameters right, then don't leave the last line behind xd

Reply:
My bad